server/app/models/TrustedIntermediaryGroupModel.java

```diff
@@ -36,13 +36,22 @@ public ImmutableList<AccountModel> getTrustedIntermediaries() {
     return ImmutableList.copyOf(tiAccounts);
   }
 
+  public int getTrustedIntermediariesSize() {
```

Comment:
Call this `getMemberCount` or something similar. It'll make more sense contextually.

Reply:
renamed